explainer-server/app/views/templates/main.scala.html

```diff
@@ -42,6 +42,21 @@
           }
         })
     </script>
+    <script type="text/javascript">
+        /*
+         * Note: there doesn't seem to be a way to acheive the update with an event (for instance onkeypress) on
+         * #explainer-editor__body-wrapper__input
+         * This explains the choice of setInterval
+         */
+        function updateWordCountDisplay(){

```

Comment:
As discussed, the event handlers could not work in either scalajs or plain javascript, so I moved to setInterval in JavaScript, but I can try that in scalajs :)